src/energyplus/Test/ControllerOutdoorAir_GTest.cpp

```diff
@@ -31,41 +44,95 @@ TEST_F(EnergyPlusFixture, ForwardTranslator_ControllerOutdoorAir) {
   Model m;
   ForwardTranslator ft;
 
+  // TODO: Without a DesignDay, the translateSizingZone is never called, and in turn the Controller:MechanicalVentilation does NOT receive the DSOAs
+  // This has been the case since the first ever commit of OS SDK on github, but it is wrong IMHO.
+  DesignDay d(m);
```

Comment:
I think you're trying to keep the PR focused on the task, which I appreciate. But, should there be a separate PR to move the assignment of DSOA objects from ForwardTranslateSizingZone.cpp to ForwardTranslateControllerMechanicalVentilation.cpp? Are you thinking it is too risky and there will be unintended consequences?

Reply:
As I said in the OP:

> This is a mininal fix extracted from https://github.com/NREL/OpenStudio/pull/5384

I handle that there...

The issue is that E+ crashes when a ControllerMV uses a  DSOA:SpaceList:
* https://github.com/NREL/EnergyPlus/issues/11018